core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -298,7 +298,7 @@ private class CSMockExternalClusterManager extends ExternalClusterManager {
     ts = mock[TaskSchedulerImpl]
     when(ts.sc).thenReturn(sc)
     when(ts.applicationId()).thenReturn("appid1")
-    when(ts.applicationAttemptId()).thenReturn(Some("attempt1"))
+    when(ts.applicationAttemptId()).thenReturn(Some("1"))
```

Comment:
This test has it as a a string and not an `int` as attempt id need not be an int for non-yarn envs.

Reply:
Reverted this change.